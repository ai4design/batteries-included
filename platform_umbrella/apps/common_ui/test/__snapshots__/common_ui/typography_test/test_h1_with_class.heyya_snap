<h1 class="text-3xl text-pink-500 sm:text-5xl font-extrabold my-6 m-10">
  Test Header With Class
</h1>