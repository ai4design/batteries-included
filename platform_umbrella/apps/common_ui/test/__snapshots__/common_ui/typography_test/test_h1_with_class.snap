"<h1 class=\"text-4xl leading-7 text-pink-500 sm:text-3xl sm:truncate m-10\">\n  Test Header With Class\n</h1>"